warp/backend.go

```diff
@@ -161,6 +171,10 @@ func (b *backend) GetMessage(messageID ids.ID) (*avalancheWarp.UnsignedMessage,
 		return message, nil
 	}
 
+	if message, ok := b.offChainMessageMap[messageID]; ok {
+		return message, nil
+	}
+
```

Comment:
This doesn't work for messages containing a block hash payload. Could we add a regression test to make sure that both message types are handled correctly?

Reply:
done